tests/component/constraint/SurfacePressureConstraintTest.cpp

```diff
@@ -49,16 +51,19 @@ using std::string;
 using std::fabs;
 using std::stof;
 
+using sofa::core::objectmodel::ComponentState;
+
 namespace sofa {
 
     template <typename _DataTypes>
-    struct SurfacePressureConstraintTest : public Sofa_test<typename _DataTypes::Real>, sofa::component::constraintset::SurfacePressureConstraint<_DataTypes>
+    struct SurfacePressureConstraintTest : public Sofa_test<typename _DataTypes::Real>, SurfacePressureConstraint<_DataTypes>
     {
 
+//        using SurfacePressureConstraint<_DataTypes>::m_componentstate;
```

Comment:
Same here...

Reply:
I will try this branch because just adding the line in master compile without problems.